cmake/projects/OpenSSL/schemes/url_sha1_openssl_macos.cmake.in

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+include(ExternalProject) # ExternalProject_Add
+
+# Scheme for download and install OpenSSL macOS library
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+
+include(hunter_dump_cmake_flags)
+include(hunter_status_debug)
+include(hunter_test_string_not_empty)
+include(hunter_unsetvar)
+include(hunter_user_error)
+
+hunter_status_debug("Scheme: url_sha1_openssl_macos")
+
+# Check preconditions
+hunter_test_string_not_empty("@HUNTER_SELF@")
+hunter_test_string_not_empty("@HUNTER_EP_NAME@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_URL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SHA1@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_DOWNLOAD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SOURCE_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_INSTALL_PREFIX@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_LICENSE_DIR@")
+hunter_test_string_not_empty("@HUNTER_GLOBAL_SCRIPT_DIR@")
+hunter_test_string_not_empty("@HUNTER_TLS_VERIFY@")
+
+# Supported macOS configurations
+# darwin-i386-cc, darwin-ppc-cc, darwin64-ppc-cc, darwin64-x86_64-cc
+
+hunter_unsetvar(ssl_input_libraries)
+hunter_unsetvar(crypto_input_libraries)
+hunter_unsetvar(ssl_input_shared_libraries)
+hunter_unsetvar(crypto_input_shared_libraries)
+
+set(configure_command "./Configure")
+
+# Replace spaces with ; to make a list.
+string(REPLACE " " ";" configure_architectures "@CMAKE_OSX_ARCHITECTURES@")
```

Comment:
`CMAKE_OSX_ARCHITECTURES` should be a CMake ;-list, hence this one not needed.

Usage examples:
* https://github.com/Kitware/CMake/blob/9a52b80d066f6c782a28878d3b28ae458f9331b3/Tests/RunCMake/XcodeProject/XcodeInstallIOS.cmake#L9
* https://github.com/Kitware/CMake/blob/9a52b80d066f6c782a28878d3b28ae458f9331b3/Tests/BuildDepends/Project/CMakeLists.txt#L22